ui/src/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+import CheckBoxIcon from "@mui/icons-material/CheckBox";
+import CheckBoxOutlineBlankIcon from "@mui/icons-material/CheckBoxOutlineBlank";
+import IconButton from "@mui/material/IconButton";
+
+export type CheckboxProps = {
+  checked?: boolean;
+  onChange?: () => void;
+  size?: "small" | "medium" | "large";
+  fontSize?: "small" | "medium" | "large" | "inherit";
+};
+
+export default function Checkbox(props: CheckboxProps) {
```

Comment:
opt: personal style preference, but destructuring the props here can be nice, i.e. `Checkbox({checked, onChange, size, fontSize}: CheckboxProps)`

Reply:
Done.